mkproj

```diff
@@ -0,0 +1,4 @@
+type = plugin
+repository = https://github.com/nju33/fish-age
+maintainer = nju33
+description = To get next version from git tag (e.g. v1.0.0)
```

Comment:
I believe this was mistakenly added

Reply:
Oh man,
I copied an existing file so I could change it with the correct info. Sorry!!